web/src/Routes.tsx

```diff
@@ -13,34 +13,22 @@ import ScaffoldLayout from 'src/layouts/ScaffoldLayout'
 
 import { useAuth } from './auth'
 import NavigationLayout from './layouts/NavigationLayout/NavigationLayout'
+import ForgotPasswordPage from './pages/ForgotPasswordPage/ForgotPasswordPage'
+import LoginPage from './pages/LoginPage/LoginPage'
+import SignupPage from './pages/SignupPage/SignupPage'
```

Comment:
You don't need to import those. they are auto-imported by redwood. If you IDE does not pick that up, you should select the typescript version to use the one from the workspace.

Reply:
Okay